estimatey-api/Estimatey.Infrastructure/DevOps/LinksSynchronizationService.cs

```diff
@@ -28,6 +29,16 @@ internal class LinksSynchronizationService
     }
 
     public async Task SyncProject(ProjectEntity project)
+    {
+        var workItemLinksBatch = await SyncProjectInternal(project);
+
+        if (!workItemLinksBatch.IsLastBatch)
+        {
+            await SyncProject(project);
```

Comment:
Noticed in `WorkItemSynchronizationService` you have a `Thread.Sleep(100)`. Do you want to do that here too?